bokehjs/test/unit/core/util/array.ts

```diff
@@ -109,4 +109,42 @@ describe("core/util/array module", () => {
     expect(array.split([1, null, 2], null)).to.be.equal([[1], [2]])
     expect(array.split([0, 1, null, 2, null, 3, 4, 5], null)).to.be.equal([[0, 1], [2], [3, 4, 5]])
   })
+
+  it("linspace should return an array of a given length between two given numbers", () => {
+    expect(array.linspace(2, 3, 5)).to.be.equal([ 2, 2.25, 2.5, 2.75, 3 ])
+    expect(array.linspace(2, 3).length).to.be.equal(100)
+    expect(array.linspace(2, 3, 0)).to.be.equal([])
+    expect(array.linspace(2, 3, NaN)).to.be.instanceof(RangeError)
+    expect(array.linspace(2, 3, null)).to.be.equal([null])
+    expect(array.linspace(NaN, NaN).length).to.be.equal(100)
+  })
+
+  it("transpose should return the given array with transposed axes", () => {
+    expect(array.transpose([[1, 2], [3, 4]])).to.be.equal([[1, 3], [2, 4]])
+    expect(array.transpose([[1, 2, 3, 4], [5, 6, 7, 8]])).to.be.equal([[1, 5], [2, 6], [3, 7], [4, 8]])
+    expect(array.transpose([[1, 2, 3, 4], [5, 6, 7]])).to.be.equal([[1, 5], [2, 6], [3, 7], [4, undefined]])
+    expect(array.transpose([[1, null, 3, undefined], [NaN]])).to.be.equal([[1, NaN], [null, undefined], [3, undefined], [undefined, undefined]])
+  })
+
+  it("remove_at should remove an item from an array at the specified index", () => {
+    expect(array.remove_at([1, 2, 3, 4], 1)).to.be.equal([1, 3, 4])
+    expect(array.remove_at([1, 2, 3, 4], 5)).to.be.equal([1, 2, 3, 4])
+    expect(array.remove_at([NaN], NaN)).to.be.equal([])
```

Comment:
The second argument is supposed to be an integer index. The type system doesn't distinguish integers from floats, so technically this is possible, but it's hardly practical. If you want to be strict about such cases, then I would add an assertion (`assert(isInteger(i) && i >= 0)`) to `remove_at()`, and change this test to catch `AssertionError`.